src/components/flag/Flag.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+import * as Flags from "./flags"
+import { SFlags } from "./styles/SFlags";
+import { IFlag } from "./IFlag";
+
+const Flag: FC<IFlag> = ({ countryCode, size }) => {
+  const Flag = countryCode && Flags[countryCode];
```

Comment:
This needs to be more defensive and show an empty flag icon in case the code is invalid. Otherwise it breaks.

<img width="1443" alt="Screen Shot 2021-11-22 at 5 58 51 PM" src="https://user-images.githubusercontent.com/642960/142935627-7d5467e9-50d1-4b88-998f-f1911b51cf9d.png">

Kind of, if (!Flag) return <EmptyFlag />